aopy/preproc/bmi3d.py

```diff
@@ -504,9 +507,26 @@ def find_laser_stim_times(laser_event_times, laser_event_widths, laser_event_pow
     # Correct the widths and powers with the given thresholds
     corrected_widths, widths_above_thr = validate_measurements(laser_event_widths, adjusted_widths, thr_width)
     corrected_powers, powers_above_thr = validate_measurements(laser_event_powers, adjusted_powers, thr_power)
+
+    if debug:
+        print(f"BMI3D recorded {len(laser_event_times)} stims")
+        print(f"Laser sensor crossed threshold {len(laser_sensor_times)} times")
+        if len(laser_event_times) == len(laser_sensor_times):
+            print(f"Average difference: {np.mean(laser_event_times - laser_sensor_times):.4f} s")
```

Comment:
If these are not equal should something be printed? Something telling the user to check their laser threshold?